Week9Serialization/Week9SerializationTest/UnitTest1.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Week9Serialization;
+
+namespace Week9SerializationTest
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
The exercise requirements specified that you should have 3 test classes - one for  each of your serializer implementations.